testharness/src/test/java/bio/terra/landingzone/db/DeployedLandingZoneDaoTest.java

```diff
@@ -81,12 +127,61 @@ public void deleteLandingZoneSuccess() {
             VERSION,
             DISPLAY_NAME,
             DESCRIPTION,
-            properties);
+            properties,
+            SUBSCRIPTION,
+            TENANT);
     landingZoneDao.createLandingZone(lz);
 
     assertTrue(landingZoneDao.deleteLandingZone(expectedLzId));
   }
 
+  @Test
+  public void getLandingZoneList_Success() {
+    UUID expectedLzId = UUID.randomUUID();
+    try {
+      LandingZone lz =
+          new LandingZone(
+              expectedLzId,
+              RESOURCE_GROUP,
+              DEFINITION,
+              VERSION,
+              SUBSCRIPTION,
+              TENANT,
+              Optional.of(DISPLAY_NAME),
+              Optional.of(DESCRIPTION),
+              properties);
+
+      landingZoneDao.createLandingZone(lz);
+
+      List<LandingZone> noRecords =
+          landingZoneDao.getLandingZoneList(SUBSCRIPTION, TENANT, "res_group");
+      assertEquals(Collections.EMPTY_LIST, noRecords);
+      List<LandingZone> noRecords1 =
+          landingZoneDao.getLandingZoneList("sub", TENANT, RESOURCE_GROUP);
+      assertEquals(Collections.EMPTY_LIST, noRecords1);
+
+      List<LandingZone> lzRecords =
+          landingZoneDao.getLandingZoneList(SUBSCRIPTION, TENANT, RESOURCE_GROUP);
+
+      assertEquals(1, lzRecords.size());
```

Comment:
nit: We should use hamcrest matchers instead of jupiter, for consistency's sake.

Reply:
Test harness and mock tests all are using jupiter matchers. 
Not changing them now - I prefer to have this as a task and fix as  in priority order.